octopusdeploy_framework/schemas/tag_set.go

```diff
@@ -74,10 +74,38 @@ func GetTagSetDataSourceSchema() datasourceSchema.Schema {
 				Optional().
 				Description("A filter to specify the number of items to take (or return) in the response.").
 				Build(),
-			"tag_sets": datasourceSchema.ListAttribute{
+			"tag_sets": datasourceSchema.ListNestedAttribute{
 				Computed:    true,
```

Comment:
Are the `Optional: false` required? I didn't think so but if they are this is missing an Optional: false.

Reply:
I was just copying Matts original PR, will add Optional in here